nx/mix.exs

```diff
@@ -38,7 +38,7 @@ defmodule Nx.MixProject do
 
   defp deps do
     [
-      {:complex, "~> 0.4.0"},
+      {:complex, "~> 0.4.0", path: "../../complex"},
```

Comment:
This needs to change with a ~> 0.4.1 release for complex due to some needed fixes in https://github.com/elixir-nx/complex/pull/20